Slim/Player/Protocols/HTTP.pm

```diff
@@ -696,8 +696,7 @@ sub getMetadataFor {
 	if ( my $currentTitle = Slim::Music::Info::getCurrentTitle( $client, $url ) ) {
 		my @dashes = $currentTitle =~ /( - )/g;
 		if ( scalar @dashes == 1 ) {
-			($artist, $title) = split / - /, $currentTitle;
-		}
+			($artist, $title) = $currentTitle =~ /(.+)[ ]+-[ ]+(.+)/s;
```

Comment:
Why did you decide to change from splitting to applying a regex?

Reply:
Don't read to much wisdom into that. I felt I could get the split and space removal in one line, rather than removing space after split ... There might be a better solution to extra all (title and option artist) in one regex, but that's beyond my abilities